lib/enkf/tests/enkf_analysis_config.cpp

```diff
@@ -131,6 +131,12 @@ void test_min_realizations_percent() {
     int min_realizations_expected_needed              = 8;
     test_min_realizations(num_realizations_str, min_realizations_str, min_realizations_expected_needed);
   }
+  {
```

Comment:
This seems like unnecessary scoping, but it is follows the style of the file at least ;)